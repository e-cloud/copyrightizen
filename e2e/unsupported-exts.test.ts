import { runCopyrightizen } from "./runCopyrghtizen";
import { copyReferenceDirectory } from './utils'
import { compare } from 'dir-compare';

describe('Unsupported extensions tests', () => {
  test('it should not change anything', async () => {
    const { tmpDir, refDir } = await copyReferenceDirectory('unsupported-exts')
    const result = await runCopyrightizen(tmpDir);

    const compareResult = await compare(refDir, tmpDir, { compareContent: true, noDiffSet: true })

    expect(result.status).toEqual(0)
    expect(compareResult.same).toEqual(true)
    expect(compareResult).toMatchSnapshot()
  })
})
