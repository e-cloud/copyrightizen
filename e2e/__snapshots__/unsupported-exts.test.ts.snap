// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unsupported extensions tests it should not change anything 1`] = `
Object {
  "differences": 0,
  "differencesDirs": 0,
  "differencesFiles": 0,
  "distinct": 0,
  "distinctDirs": 0,
  "distinctFiles": 0,
  "equal": 4,
  "equalDirs": 0,
  "equalFiles": 4,
  "left": 0,
  "leftDirs": 0,
  "leftFiles": 0,
  "right": 0,
  "rightDirs": 0,
  "rightFiles": 0,
  "same": true,
  "total": 4,
  "totalDirs": 0,
  "totalFiles": 4,
}
`;
